src/obfl-specification.html

```diff
@@ -1829,6 +1879,14 @@ <h4 id="L987" class="include">on-volume-end</h4>
           <dd class="markup note">This attribute is only available in the context of a toc-sequence.</dd>
       </dl>
     </dd>
+  <dt>Variables</dt>
+    <dd><dl>
+        <dt>started-volume-number</dt>
+          <dd>The variable holding the number of the started volume. It can be
```

Comment:
"of the started volume" -> should be "of the _ended_ volume", although that sounds a bit strange. But you get my point.

Reply:
Done.